### ISSUE TITLE
5.17.2 Commands auto-tab-complete

### ISSUE THREAD

**Issue by x1p** [BenCodez/VotingPlugin#479]
In the stable 5.17.1 version the av... commands were not visible for players without permission.
In the stable 5.17.2 these av... commands reappeared again.
It seemed logical to me that this config option should hide it.
```
# Load command aliases, such as /avgui
# Requires restart to take effect
LoadCommandAliases: false
```
But it does not seem to be any difference.
What am I doing wrong here?

**Comment by BenCodez** [comment]
Because I removed the commands and so there is no permission check on a few commands even tho they are stilled registered, working on it.

**Comment by BenCodez** [comment]
Fixed in 5.17.3
